apis/core/bootstrap/shapes/cube-project.ts

```diff
@@ -1,10 +1,10 @@
-import { cc } from '@cube-creator/core/namespace'
+import { cc, shape } from '@cube-creator/core/namespace'
 import { hydra, rdfs, sh } from '@tpluscode/rdf-ns-builders'
 import { turtle } from '@tpluscode/rdf-string'
 
 export const CubeProjectShape = turtle`
-<shape/cube-project/create> {
-  <shape/cube-project/create> a ${sh.NodeShape}, ${hydra.Resource} ;
+${shape('cube-project/create')} {
```

Comment:
Is it possible to somehow use the predefined strings from '@cube-creator/core/namespaces/shapes.ts'. here

Reply:
Precisely, the type declaration prevents strings outside of those declared in `shapes.ts`